packages/algorand-js/tsconfig.json

```diff
@@ -1,10 +0,0 @@
-{
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "outDir": "./build"
-  },
-  "include": [
-    "src",
-    "../../@types"
-  ]
-}
```

Comment:
What was different before? It wasn't a symlink?

Reply:
yes